regression-test/suites/load_p0/stream_load/test_stream_load.groovy

```diff
@@ -1575,5 +1575,125 @@ suite("test_stream_load", "p0") {
     } finally {
         sql """ DROP TABLE IF EXISTS ${tableName19} FORCE"""
     }
+
+    // load with enable_pipeline_load = true
+    def config_row = sql """ ADMIN SHOW FRONTEND CONFIG LIKE 'enable_pipeline_load'; """
+    String old_value = config_row[0][1]
+    sql """ ADMIN SET FRONTEND CONFIG ("enable_pipeline_load" = "true"); """
```

Comment:
do not do this on concurrent test. it will influence other cases. please use a new testcase with `nonConcurrent` label.

Reply:
Understood